cli/args/mod.rs

```diff
@@ -796,11 +796,18 @@ impl CliOptions {
       } else {
         None
       };
+    let parse_options = deno_config::ParseOptions {
+      include_task_comments: matches!(
+        flags.subcommand,
+        DenoSubcommand::Task(..)
```

Comment:
FWIW, this could be further optimized to `DenoSubcommand::Task(TaskFlags { task: None, .. })`.

Reply:
I thought about that, but we also print available tasks when you try to run a task that doesn't exist. I guess we could just not show the comments (or reparse the config?) but seemed more trouble than it's worth